dotnet/src/SemanticKernel.Core/KernelExtensions.cs

```diff
@@ -363,6 +364,42 @@ public static ISKPlugin ImportPluginFromObject(this Kernel kernel, object target
             description));
     #endregion
 
+    #region Kernel.Data helper methods
+
+    private const string KernelCultureDataKey = "kernel.culture";
+
+    /// <summary>
+    /// Associate a culture with the kernel.
+    /// </summary>
+    /// <param name="kernel">The kernel.</param>
+    /// <param name="culture">The culture.</param>
+    public static void SetCulture(this Kernel kernel, CultureInfo culture)
+    {
+        Verify.NotNull(kernel);
+        Verify.NotNull(culture);
+
+        kernel.Data[KernelCultureDataKey] = culture;
```

Comment:
I don't think we should store this in the dictionary. Having reserved keys like this that anyone can stomp over (accidentally or on purpose) just leads to problems.  We can just make this a private field.

Reply:
make sense.